docs/root/configuration/http/http_filters/admission_control_filter.rst

```diff
@@ -110,6 +121,8 @@ The above configuration can be understood as follows:
   window.
 * HTTP requests are considered successful if they are 1xx, 2xx, 3xx, or a 404.
 * gRPC requests are considered successful if they are OK or CANCELLED.
+* Requests will never be rejeted from this filter if the RPS is lower than 5.
+* Rejection probability will never exceed 80% even if the failure rate is 100%.
```

Comment:
If this is true and these values are clamped, can you make this more clear in the API docs? If it's not true can you make it more clear here?

Reply:
This isn't true all the time-- it's just explaining how to read the configuration above. Github cut out this line just above the additions:
```
The above configuration can be understood as follows:
```

Reply:
Ah OK sorry, I missed that, thanks.